package.json

```diff
@@ -46,38 +46,45 @@
     }
   },
   "dependencies": {
-    "@types/compression": "^1.7.3",
-    "@types/express-fileupload": "^1.4.1",
```

Comment:
After serious consideration I decided to revert these two back until the next round of refactoring when I come up with a better solution regarding `compression` and `express-fileupload` dependencies.